src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -426,9 +426,9 @@ private void ResetForNewDocument()
             foreach (var ws in Workspaces.OfType<HomeWorkspaceModel>())
             {
                 foreach (var node in ws.Nodes)
-                    node.ForceReExecuteOfNode = true;
+                    node.MarkNodeAsModified();
```

Comment:
Prior to Steve's changes, the `RequiresRecalc` flag was set to `true`, which is why I let it be. However, I'll make the change as suggested. Don't see any harm in doing so.